cmd/dlt/idp/cmd.go

```diff
@@ -120,7 +121,13 @@ func run(_ *cobra.Command, argv []string) {
 		reporter.Errorf("Failed to get identity provider '%s' for cluster '%s'", idpName, clusterKey)
 		os.Exit(1)
 	}
-
+	if ocm.IdentityProviderType(idp) == ocm.HTPasswdIDPType {
+		_, existingUserList := idpPack.FindExistingHTPasswdIDP(cluster, ocmClient)
+		if idpPack.HasClusterAdmin(existingUserList) {
+			reporter.Warnf("The cluster-admin user is contained in the HTPasswd IDP. Deleting the IDP will " +
```

Comment:
Can a use choose yes or no after this warning ?

Reply:
Absolutely, I posted an output example in the MR description.